### DISCUSSION TITLE
Just submitted a PR

### DISCUSSION THREAD

**Discussion by fabriziogianni7** [Cyfrin/foundry-full-course-cu#1585] [General]
Hey guys,
Just submitted this PR to use in-line config for invariant tests.
https://github.com/Cyfrin/foundry-defi-stablecoin-f23/pull/70

I discovered that foundry [now allow this possibility](https://book.getfoundry.sh/reference/config/inline-test-config) and I decided to contribute!
Please, feel free to comment and/or request changes! will be more than glad :)

**Comment by EngrPips** [comment]
Hello @fabriziogianni7 , Thanks you so much for your contribution and your submission would be attended to as soon as possible

**Comment by PatrickAlphaC** [comment]
Brilliant! I'll take a look this week
